libraries/chain/include/graphene/chain/protocol/sidechain_address.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include <graphene/chain/protocol/base.hpp>
+
+#include <graphene/peerplays_sidechain/defs.hpp>
+
+namespace graphene { namespace chain {
+
+    struct sidechain_address_add_operation : public base_operation
+    {
+        struct fee_parameters_type { uint64_t fee = 0; };
+
+        asset fee;
+        account_id_type sidechain_address_account;
+        graphene::peerplays_sidechain::sidechain_type sidechain;
+        string address;
+        string private_key;
```

Comment:
What's stopping anyone from listening to peerplays blocks and retrieve the private key and use it when a deposit is made on bitcoin?

Reply:
I have this in mind, I put it here more like a placeholder.